tox.ini

```diff
@@ -71,19 +69,15 @@ setenv =
     dev: WEBSOCKET_URL = {env:WEBSOCKET_URL:ws://localhost:5001/ws}
     OBJC_DISABLE_INITIALIZE_FORK_SAFETY = YES
 deps =
-    tests: -r requirements/tests.txt
-    functests: -r requirements/functests.txt
-    lint: -r requirements/lint.txt
-    {format,checkformatting}: -r requirements/format.txt
-    coverage: -r requirements/coverage.txt
-    {docs,checkdocs}: -r requirements/docs.txt
-    dev: -r requirements/dev.txt
-    dockercompose: -r requirements/dockercompose.txt
-    {env:EXTRA_DEPS:}
+    pip-tools
 whitelist_externals =
+    true
     {dev,tests,functests}: sh
+list_dependencies_command = true
```

Comment:
tox does this kind of annoying thing where every time you run it it prints out a listing of the entire contents of your venv:

```terminal
$ tox -e format
format installed: black==22.3.0,click==8.1.3,isort==5.10.1,mypy-extensions==0.4.3,pathspec==0.9.0,pep517==0.12.0,pip-tools==6.6.2,platformdirs==2.2.0,tomli==2.0.0,typing-extensions==3.10.0.1
...
```

For large venvs this can be really long and (unlike in this GitHub comment) in your terminal it'll wrap over many lines rather than scrolling.

tox-pip-sync contains [code to remove this](https://github.com/hypothesis/tox-pip-sync/blob/c93d6c0d303d399c33f7bd634d2546e0c6d5edc8/src/tox_pip_sync/__init__.py#L47-L49). This both removes the annoying noise and also speeds tox up significantly.

You can get rid of the listing (and much else) by running tox with `-q` but that doesn't seem to save any time: it looks like tox is still computing the listing, just not logging it.

But `list_dependencies_command = true` in `tox.ini` gets rid of the listing by actually preventing tox from computing it in the first place. This saves about 300ms second of execution time on my machine. Note that `true` here isn't a boolean value, it's the UNIX command `true` (which just exits with `0`).

This isn't ideal: tox is still shelling out to `true` which still takes some time. It'd be ideal if there were some way to prevent tox from listing the dependencies at all. Possibly a standalone plugin for this is needed to do only this in the same way that tox-pip-sync does.